segfault in reporter.go

given this docker-compose.yml (see https://github.com/jbrekelmans/kube-compose-demo/tree/f2103d2e519c39f66d0dd2355fa184b8980e0d22):
```
version: '2.3'
services:
  backend:
    environment: ["AUTH_ID", "AUTH_TOKEN", "PORT=8081"]
    healthcheck: &healthcheck
      test: ["CMD-SHELL", "exec curl --fail http://localhost:$$PORT/"]
      retries: 3
      interval: 3s
    image: backend:latest
    ports: ["8081:8081"]

  frontend:
    depends_on: &pre-reqs
      backend:
        condition: service_healthy
    environment: ["PORT=3000"]
    healthcheck: *healthcheck
    image: frontend:latest
    ports: ["3000:3000"]

  test-runner:
    image: test-runner:latest
    entrypoint: ["/mnt/run-tests.sh"]
    depends_on: *pre-reqs
    volumes: ["./run-tests.sh:/mnt/run-tests.sh:ro"]

x-kube-compose:
  cluster_image_storage:
    type: 'docker'
  volume_init_base_image: ubuntu:latest

```
when we run the following, the second command produces a segfault
```
$ kube-compose up backend frontend -d
$ kube-compose up test-runner
service     │ status 
service     │ status 
────────────┼────────
backend     │ waiting
test-runner │ waiting
ose#limitations) 
DEBU all depends_on conditions satisfied           service=backend
DEBU k8s service backend-build1 already exists     service=backend
DEBU k8s service frontend-build1 already exists    service=frontend
INFO waiting for cluster IP assignment (2/2)      
DEBU pod backend-build1 already exists             service=backend
DEBU created pod backend-build1                    service=backend
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x1bfb6eb]

goroutine 1 [running]:
github.com/kube-compose/kube-compose/internal/pkg/progress/reporter.(*Row).RemoveStatus(0x0, 0x2a98480, 0xc0005af900)
	/home/travis/gopath/src/github.com/kube-compose/kube-compose/internal/pkg/progress/reporter/reporter.go:451 +0x2b
github.com/kube-compose/kube-compose/internal/app/up.(*upRunner).setAppMaxObservedPodStatus(0xc0002b6780, 0xc000144c60, 0x2)
```

--- a/internal/app/up/up.go
+++ b/internal/app/up/up.go
@@ -915,11 +915,13 @@ func (u *upRunner) updateAppMaxObservedPodStatus(pod *v1.Pod) error {
 	}
 	s, err := parsePodStatus(pod)
 	if err != nil {
-		app.reporterRow.AddStatus(&reporter.Status{
-			Text:      "\x1b[31merror\x1b[0m 💣💣", // bomb+bomb
-			TextWidth: 10,
-			Priority:  4,
-		})
+		if app.reporterRow != nil {
+			app.reporterRow.AddStatus(&reporter.Status{
+				Text:      "\x1b[31merror\x1b[0m 💣💣", // bomb+bomb
+				TextWidth: 10,
+				Priority:  4,
+			})
+		}
 		return err
 	}
 
@@ -931,12 +933,14 @@ func (u *upRunner) updateAppMaxObservedPodStatus(pod *v1.Pod) error {
 
 func (u *upRunner) setAppMaxObservedPodStatus(app *app, s podStatus) {
 	app.maxObservedPodStatus = s
-	switch {
-	case s == podStatusStarted:
-		app.reporterRow.AddStatus(reporter.StatusRunning)
-	case s >= podStatusReady:
-		app.reporterRow.RemoveStatus(reporter.StatusRunning)
-		app.reporterRow.AddStatus(reporter.StatusReady)
+	if app.reporterRow != nil {
+		switch {
+		case s == podStatusStarted:
+			app.reporterRow.AddStatus(reporter.StatusRunning)
+		case s >= podStatusReady:
+			app.reporterRow.RemoveStatus(reporter.StatusRunning)
+			app.reporterRow.AddStatus(reporter.StatusReady)
+		}
 	}
 	app.newLogEntry().Debugf("pod status %s", &app.maxObservedPodStatus)
 }

